src/schema/v2/me/collection.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  GraphQLFieldConfig,
+  GraphQLObjectType,
+  GraphQLNonNull,
+  GraphQLString,
+  GraphQLBoolean,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import { InternalIDFields } from "../object_identification"
+
+export const Collection: GraphQLFieldConfig<any, ResolverContext> = {
+  description: "A collection belonging to the current user",
+  type: new GraphQLObjectType<any, ResolverContext>({
+    name: "Collection",
+    description: "A collection of artworks",
+    fields: () => ({
+      ...InternalIDFields,
+      name: {
+        type: new GraphQLNonNull(GraphQLString),
+        description:
+          "Name of the collection. Has a predictable value for 'standard' collections such as Saved Artwork, My Collection, etc. Can be provided by user otherwise.",
+      },
+      default: {
+        type: new GraphQLNonNull(GraphQLBoolean),
+        description:
+          "True if this is the default collection for this user, i.e. the default Saved Artwork collection.",
+      },
+      saves: {
+        type: new GraphQLNonNull(GraphQLBoolean),
+        description:
+          "True if this collection represents artworks explicitly saved by the user, false otherwise.",
+      },
```

Comment:
**suggestion**: Do we have any immediate user-facing need for either `default` or `saves` attributes on this new type? Not sure how they would surface in the UI at the moment. Might be easy to leave them off for now and introduce either when we can quickly associate with a UI need.

Based on what I think is included in the [JSON representation at the moment](https://github.com/artsy/gravity/blob/main/app/models/domain/collection.rb#L172-L194), the following attributes seem immediately useful:
- name
- internalID
- image - via full Image type supporting cropped/resized
- Maybe artworksCount - not sure if we'd prefer to get this from the artworksConnection.